sdk/identity/identity-common/test/msalTestUtils.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This is the utility file for the tests that the plugin packages depend on. I had to do that to silence the build errors that I was receiving because of the utils for tests and sources from plugins